.circleci/config.yml

```diff
@@ -8,17 +8,16 @@ jobs:
       - run: .circleci/clang-format-diff.sh
   cross:
     docker:
-      - image: nomaddo/cross-rpi:0.1
+      - image: nomaddo/cross-rpi:0.2
     steps:
       - checkout
       - run:
           name: download header
-          command: git clone https://github.com/doe300/VC4CLStdLib.git VC4CLStdLib && cd VC4CLStdLib && cmake . && make install && cd ..
+          command: git clone https://github.com/doe300/VC4CLStdLib.git VC4CLStdLib
```

Comment:
This cannot be removed.
Using `-DVC4CL_STDLIB_HEADER_SOURCE=/home/idein/project/VC4CLStdLib/include/VC4CLStdLib.h` below makes the (installed) package look for the PCH file in `/home/idein/project/VC4CLStdLib/include/VC4CLStdLib.h.pch`, which does not exist on the target machine.